docs/docs/recipes/integrate-logto/vanilla-js/_handle_redirect.mdx

```diff
@@ -2,7 +2,7 @@ We're almost there! In the last step, we use `http://localhost:3000/callback` as
 
 ```ts
 try {
-  await logtoClient.handleSignInCallback();
+  await logtoClient.handleSignInCallback(window.location.href);
```

Comment:
Thanks. But we also need to update the doc in `versioned_docs/version-1.x` folder.